common/test/acceptance/tests/lms/test_lms_instructor_dashboard.py

```diff
@@ -361,6 +391,84 @@ def test_can_reset_attempts(self):
         exam_attempts_section.remove_student_attempt()
         self.assertFalse(exam_attempts_section.is_student_attempt_visible)
 
+    @attr('a11y')
+    def test_track_selection_a11y(self):
+        """
+        Proctored: Track selection accessibility tests
+        """
+        self.track_selection_page.a11y_audit.config.set_rules({
```

Comment:
What is actually visible when this test is run? Unlike previous classes, the setUp for this class does not appear to visit the part of the instructor dashboard related to the test. You need to log in and visit the appropriate place first.